conbench/runner.py

```diff
@@ -246,6 +246,11 @@ def record(self, result, name, error=None, publish=True, **kwargs):
             output,
         ) = self._init(kwargs)
 
+        if (tags.get("name") is not None) and (name == tags["name"]):
```

Comment:
I see that in code we now have `name == tags["name"]` and not `name != tags["name"]`.

As a user of this code I'd wonder what the `name` argument here is supposed to be. I mean `tags.get("name")` I get it now because I am starting to refine the specification around this. But the other input parameter `name` is to me at least not explained / specified. Is this also the benchmark name, really? If yes: why did / does this code even ever try to provide _two_ ways to set the benchmark name?

Reply:
I thought about this a tiny longer and while doing so I took notes in code and did this small patch: https://github.com/conbench/conbench/pull/1048